d2common/d2math/d2vector/position_test.go

```diff
@@ -1,12 +1,49 @@
 package d2vector
 
 import (
+	"math"
+	"math/rand"
 	"testing"
 )
 
-func validate(t *testing.T, original, got, want, unchanged Vector) {
+func TestEntityPosition(t *testing.T) {
```

Comment:
There is a good chance this will be removed at some point but it's useful for now.